pages/1.12/deploying-jobs/metronome-constraints/index.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: layout.pug
+navigationTitle:  Metronome Placement Constraints
+title: Metronome Placement Constraints
+menuWeight: 3
+excerpt: Understanding Metronome placement constraints
+
+enterprise: false
+---
+
+Metronome placement constraints control where jobs run. Constraints have three parts: a field name, an operator, and a value. The field can be the hostname of the agent node or any attribute of the agent node.
+
+Metronome supports a subset of Marathon operators, and supports all of Marathon's special field names.
+
+# Field Names
+
+## Agent properties as a field name
+
+Constraints with a field name beginning with `@` refer to agent properties.
+
+### Host name
+
+Entering `@hostname` as the field name matches the agent node hostnames. See `LIKE operator`, below, for a usage example.
+
+### Region and zone as field names
+
+Use the `@region` and `@zone` field names to configure [fault domain awareness and capacity extension](/1.12/deploying-services/fault-domain-awareness/). See `IS oeprator`, below, for a usage example
```

Comment:
`IS oeprator` ==> `IS operator`
I'm also not sure this should be in a code format... it is a reference to a location in the doc